testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -203,15 +210,16 @@ Any project that has the following Gems enabled:
+| Changing PhysX Dynamic Rigid Body Properties in Workflows when using *simulated* type | <ol><li>In any of the workflows with a PhysX Dynamic Rigid Body found on this page change the rigid body properties for the scenario under test.</li><li>Create an entity with a PhysX Dynamic Rigid Body component (type *simulated*) and with each of the supported [PhysX Collider shape(s)](#physx-collider-shapes) or [PhysX Shape Collider shape(s)](#physx-shape-collider-shapes), add a mesh to the entity for visibility, position the entity above the terrain/any PhysX collider and change gravity and time steps values in System and Scene Configuration settings of PhysX Configuration (PREVIEW) tool .</li><li>Enable CCD in PhysX Configuration (PREVIEW) tool and set up a scene in which two or more CCD enabled PhysX rigid bodies collide with each other by using Initial linear velocities with values greater than 50.</li><li>Enable/Disable gravity enabled toggle in PhysX Rigid Body.</li><li>Enable/Disable start asleep toggle in PhysX Rigid Body.</li><li>Set sleep threshold for PhysX Rigid body and set up a scene in which PhysX Rigid body with set sleep treshold is impacted by other PhysX Rigid body.</li><li>Run the Simulation in the Editor Game Mode or in the Game Launcher after providing any of the changes mentioned above to PhysX Rigid body entity.</li></ol> | <ul><li>Scene behaves according to the changes made to rigid body.</li><li>Time step and gravity values set in PhysX Configuration (PREVIEW) tool correctly impact behaviors of rigid bodies in a scene.</li><li>CCD enabled Rigid bodies have better hit detection during scenarios involving big velocities.</li><li>Rigid bodies with gravity enabled toggle turned off do not react to gravity set in PhysX Configuration (PREVIEW)</li><li>Rigid bodies with start asleep toggle turned on have to be woken up by a force to start simulating.</li><li>Rigid bodies with set threshold go to sleep when applied forces are lower than set threshold.</li><li>Extra care should be tested around the following Rigid Body Properties:</li><ul><li>Angular & Linear damping</li><li>COM</li><li>Mass</li><li>Angular Velocity</li><li>Compute Intertia</li></ul></ul> |
```

Comment:
same suggestion as the other; this might be easier to follow if it was a numbered set of steps.

Reply:
Same answer as the previous comment.

But on this case I'd kind of agree that 1 entry with 7 steps (plus using variants) perhaps it's too much and could be benefit from splitting, simulated rigid body component is a complex component. 

I'll pass this feedback to @FuzzyCarterAWS but I'll keep this PR focused on the current changes and not mix up with modifying the steps themselves, which can come in a follow up PR.

Thanks.